google/async_resumable_media/requests/_request_helpers.py

```diff
@@ -90,10 +95,11 @@ def _get_body(response):
         Returns:
             bytes: The body of the ``response``.
         """
-        if response._content is False:
-            response._content = response.content
-            response._content_consumed = True
-        return response._content
+
+        # TODO() Used wrapper to extract raw content
```

Comment:
todo could likely be removed here, we have an issue to look into refactoring where combined response is. and the work to remove it is a bit bigger anyway.

Reply:
Deleted the todo statement